README.md

```diff
@@ -51,3 +52,33 @@ It can also be nicely used in conjuction with libraries such as [HTTPoison](http
   end
 ```
 
+## Configuration
+
+The default configuration for this will simply log to the console, if you would like to configure it you can simply edit your `config.exs` file.
+It is worth noting that you must keep the format as `format: "$message\n"` with whichever logging backend you choose to use otherwise you will get duplication of information.
```

Comment:
Why `format: message` and why does the information get duplicated?

Reply:
How do we guarantee the messages are in a hash/json format? If a single string is used, does this get logged?

Reply:
The readme in the PR  linked in the previous comment seems to be for the logging. Is there a particular part that I should look at for the string example? The examples in the readme look like component requests but I may have missed your point.

Not sure why the datetime would be duplicated without a "\n". Did you find out why it was doing this in your investigations?